scripts/print-workspace-status.sh

```diff
@@ -4,8 +4,11 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-apiserver_builder_version="v1.16.alpha.0"
-k8s_vendor="v1.16.0"
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+VERSION_FILE=${SCRIPT_ROOT}/VERSION
+
+apiserver_builder_version=$(cat ${VERSION_FILE})
+k8s_vendor=$(cat ${VERSION_FILE})
```

Comment:
so what's the format of `k8s_vendor`? I think this may be the version of kubernetes not the version of apiserver-builder-alpha

Reply:
good catch, fixed kubernetes version